### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by saihaj** [ScribeDAO/scribedao.github.io#1]
- [ ] Setup Tailwind
- [ ] Setup MDX
- [ ] Base Components
  - [ ] Navbar
  - [ ] Footer
  - [ ] Buttons
  - [ ] Layout/Page Shell

**Comment by brandonfrulla** [comment]
Would we want to add a sort of 'Twitter feed' component to the site, so all tweets that are shared by the ScribeDAO Twitter can also be seen on the website quickly? Might make it faster for people to find summaries that interest them

**Comment by saihaj** [comment]
Yes that's the plan. There are old mock-ups for this in discord somewhere.
